EIPS/eip-1679.md

```diff
@@ -18,6 +18,10 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 - Activation: TBD
 - Included EIPs: TBD
 
+## Proposed changes
```

Comment:
Context: at the Eth Magicians Paris event this Monday I've proposed that #233 is extended with this rule. Proposed changes (which passed a certain level scrutiny and are well considered) can be listed in the work in progress meta.

Reply:
But nobody had defined what level of scrutiny an EIP must pass before being added there.